packages/shared/lib/core/wallet/utils/getOutputParameters.ts

```diff
@@ -49,7 +49,8 @@ function getAmountFromTransactionDetails(transactionDetails: NewTransactionDetai
     let rawAmount: string
     if (transactionDetails.type === NewTransactionType.TokenTransfer) {
         const asset = transactionDetails.asset
-        const nativeTokenId = asset?.id === get(selectedAccountAssets)?.baseCoin?.id ? undefined : asset?.id
+
+        const nativeTokenId = asset?.id === getCoinType() ? undefined : asset?.id
```

Comment:
is this correct? before we had `get(selectedAccountAssets)?.baseCoin?.id` amd now its `getCoinType()`, afaik they are not the same thing